Protobuild.toml

```diff
@@ -53,10 +48,6 @@ plugins = ["grpc"]
 #
 # We have a few examples to map packages from the examples.
 [packages]
```

Comment:
The package replacement still works, but gogo would not work with the latest protobuf. We may need another packages as an example.

https://developers.google.com/protocol-buffers/docs/reference/go-generated#package